snow/networking/handler/health.go

```diff
@@ -54,5 +57,24 @@ func (h *handler) networkHealthCheck() (interface{}, error) {
 		)
 	}
 
+	details["disconnectedValidators"] = h.getDisconnectedValidators()
+
 	return details, err
 }
+
+func (h *handler) getDisconnectedValidators() set.Set[ids.NodeID] {
```

Comment:
set.Set is sorted when marshalled into JSON.

Reply:
I think this is only used here rather than being a shared utility function, so IMHO it should be fine. I added cases to the existing UT to cover this.